projects/storefrontstyles/scss/components/checkout/cart/_cart-item.scss

```diff
@@ -18,7 +18,7 @@
     padding: 0;
 
     @include media-breakpoint-up(xl) {
-      min-width: 17rem;
```

Comment:
this part seems missing in the file i pulled, pls reset this file to `develop` status, and transfer all change to `calydon`